src/sass/extends/_blockquotes.scss

```diff
@@ -26,7 +26,6 @@ blockquote:before {
 2. Blockquote, mobile
 ------------------------------------------------------------------- */
 .uds-blockquote {
```

Comment:
The blockquote/testimonial element did receive some updates with this latest package update, but they appear minor in scope. Plus, the Blockquote block uses markup contained directly within the theme as opposed to this particular markup to express a blockquote.